actions/client-review-grid.mdx

```diff
@@ -27,8 +27,528 @@ Once the filter is applied, all your submitted files will be visible in the list
 
 <Frame>
   <img src="https://cdn.labellerr.com/1%20%20Documentation/dfbd5356-a81c-4bfc-bf32-e3adc2cdb0e0.webp" alt="Filtered Files List" />
+</Frame>---
+title: "Client Review Using Grid View"
+description: "Learn how clients can efficiently review and manage annotated files in Labellerr using the grid view, with quick accept or reject options."
+icon: grid-2
+---
+
+This feature allows clients to view files in a grid view, enabling users to quickly accept or reject files with just one click. Below, we provide step-by-step guidance on how to use this functionality.
+
+### 1. Access the Annotated file
+
+*   Labeling must be completed within the **Label** section.
+*   Once labeling is done, files should be submitted.
```

Comment:
same for this page

<img width="1557" height="993" alt="image" src="https://github.com/user-attachments/assets/0d8c9463-bf67-4673-b414-222269b04d0c" />